index.html

```diff
@@ -8,10 +8,30 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <nav>
-        <ul>
-            <li>Logo</li>
-        </ul>
-    </nav>
+    <div id="header-background">
+        <nav id="flex-nav">
+            <ul>
+                <li id="logo"><a href="index.html">My Logo</a></li>
+                <li id="burguer-menu-icon ">
+                    <img src="img/menu.svg" alt="burguer-menu" class="burguer-menu-vector">
+                </li>
+            </ul>
+        </nav>
+        <div id="primary-text">
```

Comment:
This part represents a generic standalone section of the document, so it is important to use `section` tag instead of a `div` tag.